src/TeamPermissions.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+/*
+ * (c) Packagist Conductors GmbH <contact@packagist.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace PrivatePackagist\ApiClient;
+
+class TeamPermissions
```

Comment:
final?

Reply:
@glaubinix do you have a preference?

Reply:
no preference really. Might make it easier down the road if we have this final

Reply:
okay, made it final :+1: